src/komponenter/avsluttModal.test.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-undef */
+import React from 'react';
+
+import AvsluttModal from './AvsluttModal';
+import { Hovedknapp, Modal, Undertittel, Normaltekst } from '../utils/navFrontend';
+
+
+let wrapper;
+
+describe(('DokumentKort Test Suite'), () => {
+  const closeModal = jest.fn();
+  beforeEach(() => {
+    wrapper = shallow(<AvsluttModal closeModal={closeModal} visModal={false} />);
+  });
+  afterEach(() => {
+    closeModal.mockClear();
+  });
+  describe('Presentasjon', () => {
+    it('somketest', () => {
+      expect(wrapper).toHaveLength(1);
+    });
+    it('mounter Modal med korrekte props', () => {
+      const component = wrapper.find(Modal);
+      expect(component).toHaveLength(1);
+      expect(component.props().isOpen).toEqual(false);
+      expect(typeof (component.props().onRequestClose)).toEqual('function');
```

Comment:
expect(component.props().onRequestClose).to.be.function
@see https://devhints.io/chai